resources/assets/styles/components/blocks/_colors.scss

```diff
@@ -0,0 +1,15 @@
+@each $name, $color in $wp-blocks-colors {
+  .has-#{$name}-color {
+    color: $color;
+  }
+
+  .has-#{$name}-background-color::before {
+    background-color: $color;
+  }
+
+  @for $i from 1 through 10 {
+    .has-background-dim.has-background-dim-#{ $i * 10 }::before {
```

Comment:
This selector doesn't actually connect the dim bg styles to the colors. Needs to be something like:

```scss
@for $i from 1 through 10 {
  .has-#{$name}-background-color.has-background-dim-#{ $i * 10 }::before {
    background-color: rgba($color, $i * 0.1) !important;
  }
}
```

Reply:
My latest commit includes this fix 👊🏼